packages/server/.env.development

```diff
@@ -1,7 +1,7 @@
 PUBLICKEY=BICM9ljt6UzPC2HsKwlGotdrOSjva59TcwEnolgjf67P8BLN-UStYY2kvTyKDT5DpqJX0sK1z5ZyCdc98_z-AjY
 PRIVATEKEY=3134tddi2nyoxnamvbImuban4fXXgiiXia4t3SGVOBA
 EMAIL=mailto:zuck@fb.com
-TWILIOACCOUNTSID=REMEMBERTHEALAMO
+TWILIOACCOUNTSID=ACREMEMBERTHEALAMO
 TWILIOAUTHTOKEN=FLYMETOTHEMOONANDLETMESINGFOREVERMORE
 TWILIOPHONENUMBER=+14204206969
-COOKIE_PASSWORD=chunguschunguschunguschunguschunguschunguschunguschunguschungus 
\ No newline at end of file
+JWT_SECRET=chunguschunguschunguschunguschunguschunguschunguschunguschungus 
```

Comment:
what's the `JWT_SECRET`?

Reply:
`JWT_SECRET` is the private key used to secure the auth cookie which is set in the user browser